bears/c_languages/CPPCleanBear.py

```diff
@@ -1,18 +1,14 @@
-import re
+from coalib.bearlib.abstractions.Linter import linter
 
-from coalib.bearlib.abstractions.Lint import Lint
-from coalib.bears.LocalBear import LocalBear
 
+@linter(executable='cppclean',
+        output_format='regex',
+        output_regex=r'.+:(?P<line>\d+):(?P<message>.*)')
+class CPPCleanBear:
+    """
+    Checks code with ``cppclean``.
+    """
 
-class CPPCleanBear(LocalBear, Lint):
-    executable = 'cppclean'
-    output_regex = re.compile(
-        r'(?P<file_name>[^,:]+):(?P<line>\d+):(?P<message>.*)')
-    use_stdout = True
-    arguments = '{filename}'
-
-    def run(self, filename, file):
-        '''
-        Checks code with `cppclean`.
-        '''
-        return self.lint(filename)
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return filename,
```

Comment:
this syntax is correct, you're creating a list first and then make a tuple out of that by iterating over it, that gives even a performance penalty